vlib/v/checker/tests/string_interpolation_wrong_fmt.out

```diff
@@ -0,0 +1,63 @@
+vlib/v/checker/tests/string_interpolation_wrong_fmt.v:3:16: error: precision specification only valid for float types 
+    1 | fn interpolate_str() string {
+    2 |     a := 'hallo'
+    3 |     x := '>${a:8.3s}<'
+      |                   ^
+    4 |     y := '${a:G}'
+    5 |     z := '${a:d}'
+vlib/v/checker/tests/string_interpolation_wrong_fmt.v:4:12: error: illegal format specifier G for type string 
```

Comment:
the convention is to use ` when quoting:
```
error: illegal format specifier `G` for type string ```

Reply:
Yes, agreed... I've added some `s - also around the type name.